src/test/scala/com/stratio/hermes/utils/HermesTest.scala

```diff
@@ -135,6 +135,33 @@ class HermesTest extends FlatSpec with Matchers {
     //scalastyle:on
   }
 
+  it should "generate valid locations: ES and US locales" in {
+
+    val hermesES = Hermes("ES")
+    hermesES.Geo.geoModel should contain (hermesES.Geo geolocation)
+
+    val hermesUS = Hermes("US")
+    hermesUS.Geo.geoModel should contain (hermesUS.Geo.geolocation)
+  }
+
+  it should "raise an exception when it gets a geolocation are empty in the locale" in {
```

Comment:
The test descripcion is not clear I'd change it to be: it should "raise a NoSuchElementException when the locale is empty"